alexkamburov/vepaks-regular-exam
# Fix test files with multiple updates

- Update 'about' test: Change heading to 'h1', adjust text to 'About'
- Modify 'add book' test: Update URL and form selector
- Revise 'book' test: Change navigation to list selection
- Update 'footer' test: Adjust copyright text
- Modify 'header' test: Change expected text, remove comment
- Update 'home' test: Adjust expected text for home page

All 6 tests passed using 4 workers.
